src/firestore/delete.ts

```diff
@@ -336,13 +346,13 @@ export class FirestoreDelete {
             queue = queue.concat(docs);
             lastDocName = docs[docs.length - 1].name;
           })
-          .catch((e) => {
+          .catch((e: unknown) => {
             logger.debug("Failed to fetch page after " + lastDocName, e);
             pageIncoming = false;
 
             fetchFailures++;
             if (fetchFailures === 3) {
```

Comment:
Nit: This could be more defensively written as:
```suggestion
            if (fetchFailures >== 3) {
```